### ISSUE TITLE
discovery: Incentives for Organization Contributions to OSS Projects and Individuals

### ISSUE THREAD

**Issue by kylemann** [thnxdev/issues#7]
_TLDR? this issue aims to explore existing or unseen organizational incentives that could be leveraged into a the existing thanks.dev product._

## Problem to Solve
Incentivizing organizations—or better understanding their existing incentives—to support open-source projects is a central focus for thanks.dev, as discussed in our initial conversation.

While the osspledge initiative effectively raises awareness and we can learn from the campaign. It's also worth considering that developer tool companies have clearer marketing incentives/upside to support OSS in this way since their customers are developers. With this in mind, how can thanks.dev appeal to non-developer-focused organizations, too? What incentives could thanks.dev build on for meaningful, ongoing support/subscriptions for open-source projects from a broader range of organizations?

> An example of this can be seen in the numerous responses here:
https://news.slashdot.org/story/24/10/25/1654257/san-francisco-billboards-call-out-tech-firms-for-not-paying-for-open-source
This is literally part of the problem. Software is not some altruistic charity. These people still need to live. We need to recognize that that we can afford to make all sides happy.
_source: https://x.com/zeeg/status/1850953749675851869_

Drawing from the above thread, you see the debate of OSS and funding continue: "people need to live...software isn't an altruistic charity" OR "it requires tech companies to feel shame for what they are doing" OR "how are they determining who gets paid, and how much?" OR  "Why ought they be compensated? They volunteered to give away their work. "

## Discovery
While there may be varied views on OSS and paying, thanks.dev is focused on the part of the marketplace where the maintainer wants to work on the project full time and the organizations that want to ensure that said project continues (e.g. is funded to keep the lights on). Let's explore how we can connect those two in the marketplace / build on existing product.

Let's explore the following questions:

- What are existing incentives that drew in organizational funding? Consider: audit of existing product and customers in thanks.dev. How did these organizations get approval? did they have internal benefits such as tax incentives?
     - Get thanks.dev in front of organizational leaders and understand their perspective/perception of the tool 
- How do organizations _outside_ the dev tooling space, perceive OSS support? Do they see it as a value-add, a risk mitigator, or something else entirely?
     - What unique incentives could resonate with these organizations that don’t see direct marketing benefits?
- How important is reputation or brand alignment with OSS to different types of organizations? 
- Could highlighting OSS’s impact on security or reliability drive engagement? How can we quantify or showcase dependency risk mitigation as a selling point?
- Are there legal or tax benefits we could leverage to make OSS support more attractive to these organizations?
- How can the visibility of their support be showcased in a way that aligns with these organizations goals (consider: internal employee engagement, corporate responsibility initiatives etc)?
- How would a contribution mechanism like thanks.dev be surfaced to organizations and/or even better what would triggers organizations to seek out something like thansk.dev?

**Comment by kylemann** [comment]
### WIP: different approaches in the ecosystem:
- https://opensourcepledge.com/
     - companies to pay a minimum of $2000 per year per full-time equivalent developer on the company's staff, to Open Source maintainers of each company's choosing. 
- https://opencoreventures.com/
     - identifies open source software projects with traction, recruits project founders and contributors, and launches new company (separate from original maintainers?) 
- https://tea.xyz/
     - Creates rewards structures for contributions  
- https://grants.gitcoin.co/
     - Facilitates grants toward projects 
- https://oss.capital/
     - venture funding for commercial open source orgs

**Comment by kylemann** [comment]
### WIP: user testing/perception about product
Find organizational leaders and get thanks.dev in front of them to learn about how they perceive it

<details><summary>p1 notes</summary>


org/user review
- loves the domain
- overall money seems nominal
- seems to be a leaderboard of open source sponsorship, "all that i can gather so far" not clear what to do
- re: the big tech donated, unsure why it's not the same for all of them. clicked a specific 
- note: 1m means different things to different cultures
- first noticed what do do at the bottom of the why section
- login was offsetting, since 
- GitHub installation doesn't make redirect
- unable to switch orgs
- missing the text for the header
     - vision should be on the marketing page
     - marketing 

- thoughts on OSS contributions
     - for non dev orgs
          - if something goes wrong with dependencies
          - companies have budget and they have goals to solve as a business. marketing is on that, second is "scortched earth" is supporting projects taht are not in your core competencies and moral standing and could be a recruiting mechanism for developers and matching the monetary one in particular. OSS may not be tax write off. or forward looking software and crowdfunding said project. crowdfunding that change   

</details>

**Comment by kylemann** [comment]
### WIP: understand incentives from orgs that have contributied 
audit existing org contributors and outline their motivations for funding
- ...

**Comment by kylemann** [comment]
### WIP: understand incentives from orgs that are not dev-tool
the incentives for dev tool orgs supporting OSS are more clear. how about software orgs in non developer tooling space
- ...

**Comment by kylemann** [comment]
## WIP: from a maintainers perspective

<details><summary>participant notes</summary>
- compared to other platforms he heard about: (changelog podcast) 
- thinks he needs to book a call 1st, user says he wouldn't book a call
- seems like some infrastructure to donate money or give money, so it's not so interesting bc there are already systems.
- (loading) i don't know why it's ready my repos
- I don't see anything with taxes (how are taxes handled, does it handle that for me?)
- unsure why the dependencies are there
- it's not clear to me (as he was toggling around), I may not need this at this time
- this and other products are already in GitHub already


</details>
